datascience-notebook/Dockerfile

```diff
@@ -11,9 +11,16 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     fonts-dejavu \
     gfortran \
+    unixodbc-dev \
+    libtool-bin \
+    autoconf \
+    automake \
     gcc && apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
+# Python pyodbc
+RUN pip install pyodbc
```

Comment:
Seems like a good idea.  Perhaps the .odbc.ini file could be prepared to use the correct credentials for the maria, mysql image.  Perhaps a base database per-populated and setup as a test odbc image preconnected to test the odbc connection.  That way the automated build might be able to run a system test using this populated data  so future changes can make sure the odbc stuff still is working.